src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/germline/CoverageModellerGermlineSparkToggleIntegrationTest.java

```diff
@@ -0,0 +1,398 @@
+package org.broadinstitute.hellbender.tools.coveragemodel.germline;
+
+import htsjdk.samtools.util.Log;
+import junit.framework.Assert;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.math3.linear.RealMatrix;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.coveragemodel.CoverageModelEMParams;
+import org.broadinstitute.hellbender.tools.coveragemodel.CoverageModelGlobalConstants;
+import org.broadinstitute.hellbender.tools.coveragemodel.nd4jutils.Nd4jApacheAdapterUtils;
+import org.broadinstitute.hellbender.tools.coveragemodel.nd4jutils.Nd4jIOUtils;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollection;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollectionUtils;
+import org.broadinstitute.hellbender.tools.exome.Target;
+import org.broadinstitute.hellbender.tools.exome.TargetTableReader;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.ContigGermlinePloidyAnnotationTableReader;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.GermlinePloidyAnnotatedTargetCollection;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.SexGenotypeDataCollection;
+import org.broadinstitute.hellbender.utils.LoggingUtils;
+import org.broadinstitute.hellbender.utils.SparkToggleCommandLineProgram;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import javax.annotation.Nonnull;
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Integration tests for {@link CoverageModellerGermlineSparkToggle}
+ *
+ * TODO github/gatk-protected issue #803 test case-sample calling on rearranged targets
+ * TODO github/gatk-protected issue #803 test concordance on parameter estimation
+ * TODO github/gatk-protected issue #803 test Spark results match local results (of course they do, but is good to test anyway...)
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public class CoverageModellerGermlineSparkToggleIntegrationTest extends CommandLineProgramTest {
```

Comment:
Personally, and @samuelklee can override me here, I like integration tests to include tests in which the test code itself writes some artificial data with obvious calls and the runs the tool on the artificial data.  That way the tet code is self-contained.